stable/chaoskube/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: chaoskube
-version: 0.14.0
+version: 0.14.1
```

Comment:
lets bump to 1.0.0 since this chart is in stable.
```suggestion
version: 1.0.0
```

Reply:
Updated